lib/trento/application/read_models/database_read_model.ex

```diff
@@ -18,6 +18,7 @@ defmodule Trento.DatabaseReadModel do
   schema "databases" do
     field :sid, :string
     field :health, Ecto.Enum, values: Health.values()
+    field :deregistered_at, :utc_datetime_usec
```

Comment:
Just to be consistent, we could put this at the bottom of the file, as we have in the sap systems projector